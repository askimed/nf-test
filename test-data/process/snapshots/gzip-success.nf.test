nextflow_process {

  name "Test process xy"

  script "./copy.nf"
  process "COPY"

  test("Should succeed because valid gz file") {

    when {
      process {
        """
        input[0] = file("${moduleTestDir}/valid.txt.gz")
        """
      }
    }

    then {
      assert process.success
      assert snapshot(process.out.gzip).match()
    }

  }

}
