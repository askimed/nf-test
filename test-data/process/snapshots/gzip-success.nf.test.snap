{
    "Should succeed because valid gz file": {
        "content": [
            [
                "output.gz:md5,725fe503106b990f4cf2a906c7219cf0"
            ]
        ],
        "meta": {
            "nf-test": "0.8.3",
            "nextflow": "23.04.3"
        },
        "timestamp": "2024-01-28T16:18:48.680744"
    }
}